tests/JAXLLoggerTest.php

```diff
@@ -33,4 +35,18 @@ public function testLog()
         uopz_backup('error_log');
         JAXLLogger::log($msg);
     }
+
+    /**
+     * @requires PHP 5.4
+     * @requires function uopz_backup
```

Comment:
This is probably not needed. This test was failing for me because `debug_backtrace()` doesn't produce expected output by the code. I think that code needs refactoring in order for these test to be useful.